package.json

```diff
@@ -63,6 +63,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-test-loader": "^2.1.0",
+    "ember-cli-tinycolor": "^1.0.0",
```

Comment:
I like this, especially the (color modifications)[https://github.com/bgrins/TinyColor#color-modification] it supports. We might use it for theming at some point.